dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/MetricsDataS3Client.java

```diff
@@ -74,17 +68,15 @@ public MetricsDataS3Client(String bucketName, URI endpointOverride) {
      * @param versionName The GA4GH ToolVersion name
      * @param platform The platform that the metrics data is from
      * @param fileName The file name to use. Should be the time that the data was submitted in milliseconds since epoch appended with '.json'
-     * @param ownerUserId The user id of the owner (user that sent the metrics data)
+     * @param ownerUserId The Dockstore user id of the owner (user that sent the metrics data)
+     * @param description An optional description for the metrics data
      * @param metricsData The metrics data in JSON format
      */
-    public void createS3Object(String toolId, String versionName, String platform, String fileName, long ownerUserId, String metricsData) {
+    public void createS3Object(String toolId, String versionName, String platform, String fileName, long ownerUserId, String description, String metricsData) {
         try {
             String key = generateKey(toolId, versionName, platform, fileName);
-            Map<String, String> metadata = Map.of(ObjectMetadata.TOOL_ID.toString(), toolId,
-                    ObjectMetadata.VERSION_NAME.toString(), versionName,
-                    ObjectMetadata.PLATFORM.toString(), platform,
-                    ObjectMetadata.FILENAME.toString(), fileName,
-                    ObjectMetadata.OWNER.toString(), String.valueOf(ownerUserId));
+            Map<String, String> metadata = Map.of(ObjectMetadata.OWNER.toString(), String.valueOf(ownerUserId),
+                    ObjectMetadata.DESCRIPTION.toString(), description);
```

Comment:
If `description`, or less likely, `metricsData` is null or empty, will the PutObjectRequest work?